pmc/test_procedures/indexing-querying.json

```diff
@@ -0,0 +1,137 @@
+{
+  "name": "indexing-querying",
+  "description": "Indexes half the data for NYC Taxis, then concurrently queries OpenSearch while indexing again.",
+  "default": false,
+  "schedule": [
+    {
+      "operation": {
+        "operation-type": "put-settings",
+        "body": {
+          "transient": {
+            "search.default_search_timeout": "{{default_search_timeout | default(-1)}}"
```

Comment:
Are we planning to set this field to other workloads as well?

Reply:
Yes, we'll add this one at a time.